plaid/accounts.go

```diff
@@ -0,0 +1,102 @@
+package plaid
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type Account struct {
+	ID      string `json:"account_id"`
+	Balance struct {
+		Available              float64 `json:"available"`
+		Current                float64 `json:"current"`
+		Limit                  float64 `json:"limit"`
+		ISOCurrencyCode        string  `json:"iso_currency_code"`
+		UnofficialCurrencyCode string  `json:"unofficial_currency_code"`
+	} `json:"balances"`
+	Mask         string `json:"mask"`
+	Name         string `json:"name"`
+	OfficialName string `json:"official_name"`
+	Subtype      string `json:"subtype"`
+	Type         string `json:"type"`
+}
+
+type AccountNumber struct {
+	Account     string `json:"account"`
+	AccountID   string `json:"account_id"`
+	Routing     string `json:"routing"`
+	WireRouting string `json:"wire_routing"`
+}
+
+type accountRequestJsonOptions struct {
+	AccountIDs []string `json:"account_ids"`
+}
+
+type accountRequestJson struct {
+	ClientID    string                    `json:"client_id"`
+	Secret      string                    `json:"secret"`
+	AccessToken string                    `json:"access_token"`
+	Options     accountRequestJsonOptions `json:"options"`
+}
+
+type accountResponseJson struct {
+	APIResponse
+	Accounts []Account `json:"accounts"`
+}
+
+// Returns the real-time balance for each of an Item's accounts.
+// See https://plaid.com/docs/api/#balance.
+func (c *Client) GetBalances(accessToken string) (accounts []Account, err error) {
```

Comment:
@michaelckelly @evanlimanto, if we aren't returning contents of the full `accountResponseJson` object, how will our clients have access to the `request_id` so that they can log appropriately? 

i think it might be worth restructuring this to always return the full object instead of hiding away the contents of `APIResponse` in the function signature.

Reply:
I've updated the code to reflect your comments @ntindall  — will squash the commits after review.

Reply:
@evanlimanto sounds good -- I think that is best.

we need to export all of the `*Json` types so that our clients can actually use the data. if we return an anonymous unexported struct, I don't think they will be able to use the client effectively... 

Wlso, we should probably just axe the Json suffix on those types.

Are we linting this repo at all? I'm pretty sure the above would have been disallowed by our internal linters.

Reply:
@ntindall Is it possible to add our internal linting to CI? I'm not too familiar with our go tools...

Reply:
@ntindall sorry I missed that; I'm not super familiar with go's exporting format yet. I'm currently running the linter on my laptop, but to set it up in the CI we would need to allow access for circle to clone the dev-tools repo right?

Reply:
fyi: plaid-python uses and `flake8 plaid` and plaid-python uses jscs/jslint. To use our suite of tools included in `glint` is much more work than these other tools.